### DISCUSSION TITLE
Strong naming?

### DISCUSSION THREAD

**Discussion by dualbootsoftware** [vlingo-net/vlingo-net-uuid#11] [Ideas]
Hi,

This library is awesome - thank you very much for your effort. I am currently trying to integrate it with an existing .NET 4.8 Framework application and was wondering if there was any chance you could add signing/strong name or allow me to submit a PR to strong name? 

Thank you again!

**Comment by tjaskula** [comment]
Hi @dualbootsoftware,

We do not have anything against strong naming. However, as the uuid assembly is used by every other vlingo-xoom assembly, we would need to make a strong naming available everywhere. If you feel like you could implementent strong naming on each assembly we would gladly take any PR.

Best,

Tomasz
